projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html

```diff
@@ -153,6 +153,7 @@
             class="link cx-action-link"
             [disabled]="quantityControl.disabled"
             (click)="removeItem()"
+            attr.aria-label="{{ 'addToCart.removeFromCart' | cxTranslate }}"
```

Comment:
Just a comment, we apply the same aria attribute in two templates in two different styles. 

- in `added-to-cart-dialog.component.html` we are using property binding `[attr.aria-label]="'addToCart.closeModal' | cxTranslate"`
- in `cart-item.component.html` we do string interppolation `attr.aria-label="{{ 'addToCart.removeFromCart' | cxTranslate }}"` 

is it possible to make it consistent across both templates?